pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
-	// Username is the username of the agent requesting a template instantiation.
+	// username uniquely identifies this user among all active users.
```

Comment:
@jim-minter said this had to start with a lower case letter at https://github.com/openshift/origin/pull/15719#discussion_r132544136

can you two reconcile ?